icu4c/source/data/BUILDRULES.py

```diff
@@ -308,6 +310,36 @@ def generate_coll_ucadata(config, glob, common_vars):
     ]
 
 
+def generate_full_unicore_data(config, glob, common_vars):
+    # The core Unicode properties files (pnames.icu, uprops.icu, ucase.icu, ubidi.icu)
+    # are hardcoded in the common DLL and therefore not included in the data package any more.
+    # They are not built by default but need to be built for ICU4J data and for getting the .c source files
+    # when updating the Unicode data.
+    # See Jitterbug 4497. (makedata.mak revision 1.117)
```

Comment:
Jitterbug is what we used before Trac. Please refer to ICU-4497.
"revision 1.117" is from cvs, pre-svn ... I would just remove the parenthesis.

Reply:
Done